nemo_curator/datasets/doc_dataset.py

```diff
@@ -155,6 +155,73 @@ def read_pickle(
             )
         )
 
+    @classmethod
+    def read_custom_data(
```

Comment:
```suggestion
    def read_custom(
```
to more closely match `read_json`, `read_parquet`, `read_pickle`?